jerry-core/ecma/builtin-objects/ecma-builtin-helpers.cpp

```diff
@@ -481,11 +482,19 @@ ecma_builtin_helper_array_concat_value (ecma_object_t *obj_p, /**< array */
  */
 uint32_t
 ecma_builtin_helper_string_index_normalize (ecma_number_t index, /**< index */
-                                            uint32_t length) /**< string's length */
+                                            uint32_t length, /**< string's length */
+                                            bool nan_to_zero) /**< whether NaN is mapped to zero (t) or length (f) */
```

Comment:
This argument modifies the behavior a bit so we need to update the function comment above.